test/hybrid2d.jl

```diff
@@ -127,44 +133,216 @@ end
 
     hv_center_space, hv_face_space = hvspace_2D((-1, 1), (-1, 1))
 
-    function rhs!(dudt, u, _, t)
-        h = u.h
-        dh = dudt.h
+    function rhs!(f)
+
+        divuf = Fields.FieldVector(h = zeros(eltype(f), hv_center_space))
+        h = f.h
+        dh = divuf.h
 
         # vertical advection no inflow at bottom
         # and outflow at top
-        Ic2f = Operators.InterpolateC2F(top = Operators.Extrapolate())
-        divf2c = Operators.DivergenceF2C(
```

Comment:
@simonbyrne , in PR #180 you mentioned to add convergence tests for `DivergenceC2F`, but I thought that here we found the problem with the boundary value of `DivergenceF2C` . Do both needs checking then?

Reply:
Yes, that is a separate issue.